ofrak_core/setup.py

```diff
@@ -54,7 +54,11 @@ def read_requirements(requirements_path):
     + read_requirements("requirements.txt"),
     extras_require={
         "docs": read_requirements("requirements-docs.txt"),
-        "test": ["ofrak_angr~=1.0", "ofrak_capstone~=1.0"]
+        "test": [
+            "importlib-resources",  # Needed because of https://github.com/redballoonsecurity/ofrak/issues/398
+            "ofrak_angr~=1.0",
+            "ofrak_capstone~=1.0",
```

Comment:
I personally don't think internal package versions should be pinned. Sometimes leads to a local package being overwritten by an older one from `pip`.

Reply:
I see your point. It is likely out of scope of this PR though.